packages/hdwallet-native/src/thorchain.ts

```diff
@@ -11,7 +11,7 @@ import * as util from "./util";
 
 const THOR_CHAIN = "thorchain-mainnet-v1";
 
-const protoTxBuilder = PLazy.from(() => import("@shapeshiftoss/proto-tx-builder"));
+const protoTxBuilder = PLazy.from(() => import("@keepkey/proto-tx-builder"));
```

Comment:
Revert me.

Reply:
shapeshift will also need to update proto-tx-builder to support maya

Reply:
nah